tests/test_endpoints.py

```diff
@@ -1439,3 +1552,24 @@ async def test_with_7_x_firmware(
             await envoy.open_dry_contact("NC1")
         with pytest.raises(EnvoyFeatureNotAvailable):
             await envoy.close_dry_contact("NC1")
+
+    if (supported_features & SupportedFeatures.ENCHARGE) and (
```

Comment:
This test is starting to get too big. We should split it up at some point

Reply:
Agree, probably time to start splitting it out into multiple files